include/SFML/Audio/SoundSource.hpp

```diff
@@ -81,7 +81,7 @@ class SFML_AUDIO_API SoundSource : AlResource
     /// \see getPitch
     ///
     ////////////////////////////////////////////////////////////
-    void setPitch(float pitch);
+    void setPitch(float pitch) const;
```

Comment:
It's odd that these setter functions don't mutate the class. I guess I'm fine making these const but it just seems weird.

Reply:
@Bromeon any insight into this and whether you think this is a good idea?

Reply:
They do mutate stuff, but not its own members, but instead OpenAL states.  
I don't think we should change the interface, as it would limit our implementation flexibility.

Reply:
I'm fine disabling that check for this file then